build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/KotlinAndroid.kt

```diff
@@ -70,6 +70,6 @@ internal fun Project.configureKotlinAndroid(
     }
 }
 
-private fun CommonExtension<*, *, *, *>.kotlinOptions(block: KotlinJvmOptions.() -> Unit) {
+fun CommonExtension<*, *, *, *>.kotlinOptions(block: KotlinJvmOptions.() -> Unit) {
```

Comment:
It can be separated into files like kotlinOptions.kt, but I'm changing it to public because it looks like the rules weren't established yet and I didn't think it was wrong to have it in Kotlin's configuration.
If you have any opinions, please let me know. 🙆‍♂️

Reply:
This makes sense for sure! We don't have any strong rules for how functions can be shared in the build logic, so I see no issue with this right now.